src/Surfnet/StepupRa/RaBundle/Controller/RaManagementController.php

```diff
@@ -91,9 +92,12 @@ public function raCandidateSearchAction(Request $request)
 
         $logger->notice(sprintf('Searching for RaCandidates within institution "%s"', $institution));
 
+        $pageNumber = $request->get('p', 1);
+        Assertion::digit($pageNumber, 'Expected page number to be an integer, got "%s"');
+
         $command                 = new SearchRaCandidatesCommand();
         $command->institution    = $institution;
-        $command->pageNumber     = $request->get('p', 1);
+        $command->pageNumber     = $pageNumber;
```

Comment:
Seems like the validation was first "in" the command, and now in the controller. Did you add and then remove the integer cast accidentally and did you mean to actually convert the page number to an integer? What does this mean for the HTTP communication? Is the status code when you pass an invalid page number the same as before (400, I presume)?